cocotb/handle.py

```diff
@@ -956,7 +956,7 @@ def SimHandle(handle, path=None):
         return obj
 
     if t not in _type2cls:
-        raise TestError("Couldn't find a matching object for GPI type %d (path=%s)" % (t, path))
+        raise NotImplementedError("Couldn't find a matching object for GPI type %d (path=%s)" % (t, path))
```

Comment:
```suggestion
        raise NotImplementedError("Couldn't find a matching object for GPI type %s(%d) (path=%s)" % (handle.get_type_string(), t, path))
```
For debugging it would be nice to have the handle address too but I don't see an easy way to grab that.